ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh

```diff
@@ -232,9 +242,32 @@ cat << EOF > ~/fetch-information.yml
     shell: kcli ssh root@${CLUSTER_NAME}-installer 'oc get node'
 EOF
 
-status=0
-ANSIBLE_STDOUT_CALLBACK=debug ansible-playbook -i $SHARED_DIR/inventory ~/ocp-install.yml -vv || status=$?
-ansible-playbook -i $SHARED_DIR/inventory ~/fetch-kubeconfig.yml -vv || true
-ANSIBLE_STDOUT_CALLBACK=debug ansible-playbook -i $SHARED_DIR/inventory ~/fetch-information.yml -vv || true
+cat << EOF > ~/check-cluster.yml
+---
+- name: Check if cluster is ready
+  hosts: hypervisor
+  gather_facts: false
+  tasks:
+
+  - name: Check if cluster is available
+    shell: kcli ssh root@${CLUSTER_NAME}-installer "oc get clusterversion -o json|jq '.items[0].status.conditions[]|select(.type==\"Available\").status'"
```

Comment:
1. it failed for me while running:
```
[kni@cnfdd1 ~]$ kcli ssh root@cnfdd3-installer "oc get clusterversion -o json|jq '.items[0].status.conditions[]|select(.type==\"Available\").status'"
jq: error: Available/0 is not defined at <top-level>, line 1:
.items[0].status.conditions[]|select(.type==Available).status                                            
jq: 1 compile error
```

2. please test it on a live environment with possible edge cases - cluster is already installed, not installed, command fails etc
if this fails, then next task won't be executed

3. It requires `jq` to be installed on all hypervisors, which should be handled separately, don't assume it's there.

Reply:
The command works when run directly in the terminal from the installer machine. It seems like there is a problem with how the command is parsed when fed through `kcli ssh`

```
[root@cnfda3-installer ~]# oc get clusterversion -o json|jq '.items[0].status.conditions[]|select(.type=="Available").status'
"False"
```

I'm trying find a way to escape them so that they'll be fed through properly.

Reply:
This task is supposed to run on hypervisor, so better to test it on it.

Reply:
The 'oc get clusterversion' command already runs on the installer vm though, isn't it less efficient to pipe that data back out to the hypervisor just to run a parser on it?

Reply:
I came up with a solution, I think... it's in the latest commit